lib/dashboard/dashboard.dart

```diff
@@ -1,51 +1,40 @@
 import 'package:google_fonts/google_fonts.dart';
 import 'package:flutter/material.dart';
+import './dashboard_card.dart';
 
-// TODO: Update this placeholder widget
 class Dashboard extends StatelessWidget {
   const Dashboard({Key key}) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
-    return Stack(
-        children: <Widget>[
-          Image.asset(
-            "images/DashboardBg.png",
-            height: MediaQuery.of(context).size.height,
-            width: MediaQuery.of(context).size.width,
-            fit: BoxFit.fill,
-          ),
-          Scaffold(
-              backgroundColor: Colors.transparent,
-              appBar: AppBar(
-                backgroundColor: Colors.transparent,
-                elevation: 0.0,
-              ),
-              body: Container(
-                child: Center(
-                  child: Column(
-                    children: buildDashboardView(context)
-                  )
-                )
-              )
+    return Scaffold(
+      backgroundColor: Colors.transparent,
+      appBar: AppBar(
+        backgroundColor: Colors.transparent,
+        elevation: 0.0,
+      ),
+      body: Container(
```

Comment:
we can probably get rid of this `AppBar` widget since it's empty and doesn't hold any title text! instead, we can add top padding/margin to the container widget to make up for the lost space